packages/cli/src/commands/hydrogen/upgrade.ts

```diff
@@ -349,17 +349,15 @@ function hasOutdatedDependencies({
   });
 }
 
-export function isUpgradeableRelease({
+function isUpgradeableRelease({
   currentDependencies,
   currentPinnedVersion,
   release,
 }: {
-  currentDependencies?: Dependencies;
+  currentDependencies: Dependencies;
   currentPinnedVersion: string;
   release: Release;
 }) {
-  if (!currentDependencies) return false;
```

Comment:
It was always returning early here because we were not passing `currentDependencies`. I'm removing this since now `currentDependencies` is a mandatory property in TS.